From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-khugepaged-unify-pmd-folio-installation-with-map_anon_folio_pmd-fix
Date: Wed Oct  8 09:10:01 PM PDT 2025

remove unneeded map_anon_folio_pmd_nopf() stub, per Wei & David

Cc: David Hildenbrand <david@redhat.com>
Cc: Dev Jain <dev.jain@arm.com>
Cc: Lance Yang <lance.yang@linux.dev>
Cc: Matthew Wilcox <willy@infradead.org>
Cc: Usama Arif <usamaarif642@gmail.com>
Cc: Wei Yang <richard.weiyang@gmail.com>
Cc: Zi Yan <ziy@nvidia.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/huge_mm.h |    5 -----
 1 file changed, 5 deletions(-)

--- a/include/linux/huge_mm.h~mm-khugepaged-unify-pmd-folio-installation-with-map_anon_folio_pmd-fix
+++ a/include/linux/huge_mm.h
@@ -640,11 +640,6 @@ static inline bool unmap_huge_pmd_locked
 	return false;
 }
 
-static inline void map_anon_folio_pmd_nopf(struct folio *folio, pmd_t *pmd,
-		struct vm_area_struct *vma, unsigned long haddr)
-{
-}
-
 #define split_huge_pud(__vma, __pmd, __address)	\
 	do { } while (0)
 
_
